osi_custom_profit_loss/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2021 - TODAY, Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Profit and Loss Account Report",
+    "version": "14.0.1.0.0",
+    "summary": "Customisation for P&L report",
+    "author": "Open Source Integrators",
+    "category": "Accounting/Accounting",
+    "maintainer": "Open Source Integrators",
+    "website": "https://github.com/ursais/osi-addons",
+    "depends": ["account_accountant"],
+    "installable": True,
+    "license": "LGPL-3",
+    "development_status": "Beta",
+    "maintainers": ["dreispt"],
+}
```

Comment:
Missing README with the documentation. Copy "configuration" and "Usage" from the design page.